app/experimenter/experiments/constants/nimbus.py

```diff
@@ -213,6 +213,13 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_FX95_DESKTOP_USERS = NimbusTargetingConfig(
+    name="Desktop Users on Fx95",
+    slug="fx95_desktop_users",
+    description=("Firefox 95 Desktop users"),
+    targeting=("version|versionCompare('95.!') == 0"),
```

Comment:
This matches exactly the version "95.!" where I believe "!" is used as a "dummy" character ("less than" `0`) to relatively compare against. If we want any 95 (e.g., including "a" and "b" and dot releases), we probably want a range, e.g., compare(95) >= 0 && compare(96) < 0

Reply:
+1, should the range be "version|versionCompare('95') >= 0 && version|versionCompare('96') < 0"

Or with dummy character  "version|versionCompare('95.!’) >= 0 && version|versionCompare('96.!’) < 0" or either works

Reply:
Oh you're right, that doesn't work.  On my local:

```js
await targetingContext.evalWithDefault("version")
"96.0b4"

await targetingContext.evalWithDefault("version|versionCompare('95.!') == 0")
false

await targetingContext.evalWithDefault("version|versionCompare('96.!') == 0")
false

await targetingContext.evalWithDefault("version|versionCompare('97.!') == 0")
false
```

Using the method @Mardak is suggesting:

```js
await targetingContext.evalWithDefault("(version|versionCompare('95.!') >= 0) && (version|versionCompare('96.!') < 0)")
false

await targetingContext.evalWithDefault("(version|versionCompare('96.!') >= 0) && (version|versionCompare('97.!') < 0)")
true

await targetingContext.evalWithDefault("(version|versionCompare('97.!') >= 0) && (version|versionCompare('98.!') < 0)")
false
```

Reply:
thanks for verifying,  patch updated with feedback